MapData.py

```diff
@@ -0,0 +1,41 @@
+import os
+import threading
+from enum import Enum
+
+import matplotlib.pyplot as plt
+from scipy.misc import imresize
+
+from detail import LOCAL
+
+class MapDataFieldNamesEnum(Enum):
+    ZOOM = 'zoom'
+    RADIUS = 'radius'
+    IMAGE = 'image'
+    MARKER_PATH = 'marker path'
+    MARKER = 'marker'
+    ANNO_BOX = 'annotation box'
+    LOCATION = 'location'
+
+class MapData:
+    def __init__(self):
+        self.lock = threading.Lock()
+
+        # Map UI attributes
+        self.mapData = {}
+        self.mapData[MapDataFieldNamesEnum.ZOOM.value] = 20
+        self.mapData[MapDataFieldNamesEnum.RADIUS.value] = 0.1
+        self.mapData[MapDataFieldNamesEnum.IMAGE.value] = None
+        self.mapData[MapDataFieldNamesEnum.MARKER_PATH.value] = os.path.join(LOCAL, "marker.png")
```

Comment:
This is only used in one place (one line below). I really dont think it deserves a place in a dictionary or even a field in a class. Maybe just a constant somewhere.